medicines/pars-upload/src/auth/authPopup.js

```diff
@@ -1,20 +1,46 @@
 import { UserAgentApplication } from 'msal'
 import { msalConfig, loginRequest } from './authConfig'
 
-export function getAccount() {
-  const myMSALObj = new UserAgentApplication(msalConfig)
-  const account = myMSALObj.getAccount()
+export async function getAccount() {
+  const msalInstance = new UserAgentApplication(msalConfig)
+  const account = msalInstance.getAccount()
 
   if (account) {
+    // TODO:
+    //
+    // Temporary hack whilst we wait for implicit granting of ID tokens to be
+    // enabled by MHRA admins so we get an access token in the proper way:
+    // https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-core#3-get-an-access-token-to-call-an-api
+
+    const token = window.sessionStorage['msal.idtoken']
```

Comment:
this only works if the setting in authConfig is set to session - if that is changed to local or cookies, this will break.